### DISCUSSION TITLE
Notification when no xp for time

### DISCUSSION THREAD

**Discussion by Yamajac** [adamk33n3r/runelite-watchdog#127] [Ideas]
Honestly half of the notifications I set up seem entirely pointless and could end up being completely replaced by a single "no xp for 3 ticks" notification. 

As a side note, would also be nice for item quantity 0 to trigger when you go from 1 to 0 of an item, and -1 to trigger on every update.

**Comment by adamk33n3r** [comment]
1. Could that be similar to using the Idle Notifier plugin's notifications? Good idea though
2. That makes sense, would be a breaking change so I'd have to figure out how to migrate people over who are using `0` for its current behavior.
